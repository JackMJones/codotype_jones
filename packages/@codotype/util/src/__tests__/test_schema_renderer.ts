import { renderSchemaJson } from "../schemaRenderer";
import { userSchema, movieSchema } from "./test_state";

// // // //

describe("/src/schemaRenderer", () => {
  it("renders JSON representation correctly", () => {
    const renderedSchema: string = renderSchemaJson({
      schema: movieSchema,
      schemas: [userSchema, movieSchema],
    });
    expect(renderedSchema).toMatchSnapshot();
  });
});
