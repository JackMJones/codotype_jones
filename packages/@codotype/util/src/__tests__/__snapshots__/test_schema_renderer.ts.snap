// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`/src/schemaRenderer renders JSON representation correctly 1`] = `
"{
  \\"name\\": \\"string\\"
  \\"email\\": \\"string\\"
}"
`;
